settings/contentstructuremenu.ini

```diff
@@ -36,9 +36,8 @@ MaxNodes=150
 #    SortBy[]
 #    SortBy[]=name/ascending
 #    SortBy[]=published/descending
-# if set to "false" - default sorting will be implemented
-# (which is defined for each node)
-SortBy=false
+# if not set, it uses the sorting defined for each node (from the Ordering tab)
```

Comment:
```suggestion
# if empty array, it uses the sorting defined for each node (from the Ordering tab)
```

Reply:
@paulortvalle Are you ok with this change above? I just want to make sure we are clear clear here ;)